repairnator/docker-images/bears-dockerimage/configure_git.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
Ok actually I propose another refactoring: most of the file in `bears-dockerimage` are the same as in `pipeline-dockerimage`. So, why not creating a folder `docker-images/shared-files` and fix their path in `Dockerfile` files? It would avoid duplication

Reply:
I tried in https://github.com/Spirals-Team/repairnator/pull/432/commits/3ec983c919140bb2e1f0210366c736217888378a

But look at the error I get: https://hub.docker.com/r/fermadeiral/repairnator/builds/blfclbmbjsyiybeu3665xjn/

Am I doing something wrong or can we not do this?

Reply:
Argh! Sorry, it's indeed not possible to do it like that :( So... let's push the files back...

Reply:
Are you sure to forget about this? I was searching and apparently it might be possible, but we would need to do `docker build` directly, and we still don't do that (at least I don't remember it from our scripts). Maybe a future work?

Reply:
Ok.